src/everest/detached/jobs/everserver.py

```diff
@@ -306,7 +306,7 @@ def main():
             optimization_callback=partial(_opt_monitor, shared_data=shared_data),
         )
         if run_model.ert_config.queue_config.queue_system == QueueSystem.LOCAL:
-            evaluator_server_config = EvaluatorServerConfig()
+            evaluator_server_config = EvaluatorServerConfig(use_ipc_protocol=False)
```

Comment:
What does this do?

Reply:
With zmq, you can opt for using IPC (inter process communication) protocol, which is faster than TCP. This of-course is possible only when using LOCAL_DRIVER. On the other hand, I don't think that it solves the issue.